samples/sample_multi_transcode/src/sample_multi_transcode.cpp

```diff
@@ -358,36 +359,28 @@ void Launcher::DoTranscoding()
 {
     mfxStatus sts = MFX_ERR_NONE;
 
-    // get parallel sessions parameters
-    mfxU32 totalSessions = (mfxU32)m_pSessionArray.size();
-    MSDKThread* pthread = 0;
-
-    for (mfxU32 i = 0; i < totalSessions; i++)
-    {
-        pthread = new MSDKThread(sts, TranscodeRoutine, (void *)m_pSessionArray[i]);
-        m_HDLArray.push_back(pthread);
```

Comment:
remove m_HDLArray completely, since you don't use it anymore

Reply:
Done